docs/source/fields.rst

```diff
@@ -5,6 +5,172 @@ Model field reference
 
 Some MongoDB-specific fields are available in ``django_mongodb.fields``.
 
+``ArrayField``
+--------------
+
+.. class:: ArrayField(base_field, size=None, **options)
+
+    A field for storing lists of data. Most field types can be used, and you
+    pass another field instance as the :attr:`base_field
+    <ArrayField.base_field>`. You may also specify a :attr:`size
+    <ArrayField.size>`. ``ArrayField`` can be nested to store multi-dimensional
+    arrays.
+
+    If you give the field a :attr:`~django.db.models.Field.default`, ensure
+    it's a callable such as ``list`` (for an empty default) or a callable that
+    returns a list (such as a function). Incorrectly using ``default=[]``
+    creates a mutable default that is shared between all instances of
+    ``ArrayField``.
+
+    .. attribute:: base_field
+
+        This is a required argument.
+
+        Specifies the underlying data type and behavior for the array. It
+        should be an instance of a subclass of
```

Comment:
I'm assuming we cannot confirm embedded models until after the embedded model PR is merged.

Reply:
I plan to merge this first, then return to EMF and decide what to do.